tests/sles4sap/sapconf.pm

```diff
@@ -119,8 +120,13 @@ sub run {
       powersave throughput-performance virtual-guest);
     # Testing 'virtual-host' on a VM isn't very useful and can lead to sporadic timeout issues
     push @tuned_profiles, 'virtual-host' unless (check_var('BACKEND', 'qemu'));
-    # Add SAP profiles depending of the OS version
-    push @tuned_profiles, is_sle('15+') ? qw(sapconf saptune) : qw(sap-ase sap-bobj sap-hana sap-netweaver);
+    # Add SAP profiles depending of the products and the OS version
+    if (check_var('SLE_PRODUCT', 'sles4sap')) {
+        push @tuned_profiles, is_sle('15+') ? qw(sapconf saptune) : qw(sap-ase sap-bobj sap-hana sap-netweaver);
+    }
+    elsif (check_var('SLE_PRODUCT', 'sles')) {
+        push @tuned_profiles, is_sle('15+') ? qw(sapconf) : qw(sap-ase sap-bobj sap-hana sap-netweaver);
```

Comment:
I will check later if it's working on SLE12 as we don't have any SLE12 tests configured for the moment, no impact here.

Reply:
This block can be written as one line:
```
push @tuned_profiles, is_sle('15+') ? (check_var('SLE_PRODUCT', 'sles4sap') ? qw(sapconf saptune) : 'sapconf') : qw(sap-ase sap-bobj sap-hana sap-netweaver);
```
However if you feel it loses too much readability, leave as is.

Reply:
> This block can be written as one line:
> 
> ```
> push @tuned_profiles, is_sle('15+') ? (check_var('SLE_PRODUCT', 'sles4sap') ? qw(sapconf saptune) : 'sapconf') : qw(sap-ase sap-bobj sap-hana sap-netweaver);
> ```
> 
> However if you feel it loses too much readability, leave as is.

I agree with you, the line is not so complicated :smiley: